### PR TITLE
[type:feature] arrange sofa common tools dependencies

### PR DESCRIPTION
**Pull request by Aias00**
arrange sofa common tools dependencies

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-bootstrap/pom.xml [hunk 1]
```diff
@@ -260,6 +260,21 @@
...
+                <exclusion>
+                    <artifactId>sofa-common-tools</artifactId>
+                    <groupId>com.alipay.sofa.common</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.alipay.sofa.common</groupId>
+            <artifactId>sofa-common-tools</artifactId>
+            <version>2.1.1</version>
```
Comment (yu199195) [inline shenyu-bootstrap/pom.xml:272]: used ${sofa-common-tools.version}?

File: shenyu-integrated-test/shenyu-integrated-test-combination/pom.xml [hunk 2]
```diff
@@ -165,8 +165,17 @@
...
+                <exclusion>
+                    <artifactId>sofa-common-tools</artifactId>
+                    <groupId>com.alipay.sofa.common</groupId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.alipay.sofa.common</groupId>
+            <artifactId>sofa-common-tools</artifactId>
+            <version>2.1.1</version>
```
Comment (yu199195) [inline shenyu-integrated-test/shenyu-integrated-test-combination/pom.xml:177]: used ${sofa-common-tools.version}?

File: shenyu-integrated-test/shenyu-integrated-test-k8s-ingress-sofa/pom.xml [hunk 3]
```diff
@@ -52,8 +52,17 @@
...
+                <exclusion>
+                    <artifactId>sofa-common-tools</artifactId>
+                    <groupId>com.alipay.sofa.common</groupId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.alipay.sofa.common</groupId>
+            <artifactId>sofa-common-tools</artifactId>
+            <version>2.1.1</version>
```
Comment (yu199195) [inline shenyu-integrated-test/shenyu-integrated-test-k8s-ingress-sofa/pom.xml:64]: used ${sofa-common-tools.version}?

File: shenyu-integrated-test/shenyu-integrated-test-sofa/pom.xml [hunk 4]
```diff
@@ -83,8 +83,17 @@
...
+                <exclusion>
+                    <artifactId>sofa-common-tools</artifactId>
+                    <groupId>com.alipay.sofa.common</groupId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.alipay.sofa.common</groupId>
+            <artifactId>sofa-common-tools</artifactId>
+            <version>2.1.1</version>
```
Comment (yu199195) [inline shenyu-integrated-test/shenyu-integrated-test-sofa/pom.xml:95]: used ${sofa-common-tools.version}?

File: shenyu-bootstrap/pom.xml (hunk 1 above)
Comment (Aias00) [reply shenyu-bootstrap/pom.xml]: yeap

**Review by yu199195 — APPROVED** [review]

**Pull request merged**